### DISCUSSION TITLE
Problems Encountered and Simplified Solutions.

### DISCUSSION THREAD

**Discussion by Amalearner** [wagtail/outreachy#64] [General]
Hello fellow Wagtail contributors and interns! This is a discussion thread that would be used by everyone as a contributor/intern. When you encounter a problem, kindly document it here and how you were able to solve it in a simplified manner.

This thread is meant to be a resource for other contributors/interns who may be encountering the same problems. By sharing your solutions here, you can help others to get past the same problems more quickly and easily.

**How to Get the Most Out of It:**

Tell Us About Your road blocks: When you encounter a problem while contributing to Wagtail, please document it here. Be sure to provide a clear and concise description of the issue you faced. This should include any relevant error messages, code snippets, etc.

Break It Down, Plain and Simple: Once you've been able to work on the problem, share the solution step by step. Explain it like you're chatting with a friend.

Chime In: We're all in this together. If something's not quite clear or you've got a different approach in mind, join the discussion. Ask questions, share ideas, and lend a hand.

Check First: Before posting a new problem, have a quick scan of the thread. Someone might have already tackled the same issue and provided a solution. It's a time-saver!

Keep It Organized: To help everyone find what they're looking for, start a new discussion for each unique problem. 

Sharing is Caring: If a particular resource—like a blog post, tutorial, or documentation—helped you solve the problem, be sure to link it. Sharing makes life easier for others.

**Comment by thibaudcolas** [comment]
Isn't this the same as https://github.com/wagtail/outreachy/discussions/30?

**Reply by Amalearner** [reply]
oh my, didn't see it. let me delete this

**Comment by thibaudcolas** [comment]
Closing in favour of #30.
